rust/test_contract_2/src/lib.rs

```diff
@@ -19,16 +22,6 @@ mod tables {
         /// The result of the calculation
         pub result: i32,
     }
-}
-
-#[psibase::service(tables = "tables")]
-#[allow(non_snake_case)]
-mod service {
-    use crate::tables::{Answer, AnswerTable};
-    use async_graphql::{connection::Connection, *};
-    use psibase::*;
-    use serde::{Deserialize, Serialize};
-    use serde_aux::field_attributes::deserialize_number_from_string;
```

Comment:
Instead of changing this test to test the new functionality, we should add a new test so we test both formats.

Reply:
There are lots of places using the old format:
```
rust/test_psibase_macros/services/add-check-init/src/lib.rs:#[psibase::service(name = "addcheckinit", tables = "service_tables")]
rust/test_subjective/src/lib.rs:#[psibase::service(name = "subjective", tables = "tables")]
```